src/main/java/org/opentripplanner/openstreetmap/issues/MalformedLevelMap.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.openstreetmap.issues;
+
+import org.opentripplanner.graph_builder.issue.api.DataImportIssue;
+import org.opentripplanner.openstreetmap.model.OSMRelation;
+
+public class MalformedLevelMap implements DataImportIssue {
```

Comment:
Can you convert this to a record?

Reply:
Sorry @leonardehrenfried this should not be a record, the relationId should not be visible outside of the class. I suggest using the Issue.issue() factory instead - and not creating an explicit class.

Reply:
I figures since I'm new to the codebase that I'd just follow along with what was already there, so I mimiced the two other DataImportIssues that was already there for OSM data.

Reply:
We discussed this at the meeting today. If you use `Issue.issue` method we can merge this. Thanks!

Reply:
Removed the MalformedLevelMap class, and switched over to Issue.issue().
Changed wording from "Malformed" to a more expected "Invalid" for the issue type while I was at it.